.eslintignore

```diff
@@ -6,3 +6,4 @@ rollup.config.js
 tsconfig.json
 # The only check that applies here is really the license header:
 .eslintrc.js
+e2e/browser/testApp
```

Comment:
@NSeydoux this is the only possible strategy I have here, as the e2e browser testApp's eslint configuration conflicts with the repos eslint config, which in turn conflicts with jest 27. I'm not sure how you were able to use Jest 27 before, as we've _never_ supported it.